content/practices/landmark-regions/landmark-regions-practice.html

```diff
@@ -175,73 +175,73 @@ <h4>HTML Techniques</h4>
 
-            <p>If the HTML <code>aside</code> element technique is not being used, use a <code>role=&quot;complementary&quot;</code> attribute to define a <code>complementary</code> landmark.</p>
+            <p>HTMLの<code>aside</code>要素のテクニックが使用されていない場合、<code>role=&quot;complementary&quot;</code>属性を使用して<code>complementary</code>ランドマークを定義します。</p>
           </section>
 
           <section>
-            <h4>Examples</h4>
+            <h4>例</h4>
 
-            <p><a href="../../patterns/landmarks/examples/complementary.html">Complementary Landmark Example</a></p>
+            <p><a href="../../patterns/landmarks/examples/complementary.html">補完ランドマークの例</a></p>
           </section>
         </section>
 
         <section id="aria_lh_contentinfo">
-          <h3 class="widget-name">Contentinfo</h3>
+          <h3 class="widget-name">コンテンツ情報</h3>
 
-          <p>A <a class="role-reference" href="#contentinfo"><code>contentinfo</code></a> landmark is a way to identify common information at the bottom of each page within a website, typically called the &quot;footer&quot; of the page, including information such as copyrights and links to privacy and accessibility statements.</p>
+          <p><a class="role-reference" href="#contentinfo"><code>contentinfo</code></a>ランドマークは、ウェブサイトの各ページの下部にある一般的な情報を識別する方法で、通常はページの「フッター」と呼ばれ、著作権やプライバシー、アクセシビリティに関する記述へのリンクなどの情報を含みます。</p>
 
           <ul>
-            <li>Each page may have one <code>contentinfo</code> landmark.</li>
+            <li>各ページは、一つの<code>contentinfo</code>ランドマークを持つことができます。</li>
 
-            <li>The <code>contentinfo</code> landmark should be a top-level landmark.</li>
+            <li><code>contentinfo</code>ランドマークは、最上位のランドマークであるべきです。</li>
 
-            <li>When a page contains nested <code>document</code> and/or <code>application</code> roles (e.g. typically through the use of <code>iframe</code> and <code>frame</code> elements), each <code>document</code> or <code>application</code> role may have one <code>contentinfo</code> landmark.</li>
+            <li>ページにネストされた<code>document</code>および/または<code>application</code>役割が含まれている場合（例：通常、<code>iframe</code>と<code>frame</code>要素の使用を通じて）、各<code>document</code>または<code>application</code>役割は、一つの<code>contentinfo</code>ランドマークを持つことができます。</li>
 
-            <li>If a page includes more than one <code>contentinfo</code> landmark, each should have a unique label (see <a href="#aria_lh_step3">Step 3</a> above).</li>
+            <li>ページに複数の<code>contentinfo</code>ランドマークが含まれている場合、それぞれに一意のラベルが必要です（上記の<a href="#aria_lh_step3">ステップ3</a>を参照）。</li>
           </ul>
 
           <section>
-            <h4>HTML Techniques</h4>
+            <h4>HTMLテクニック</h4>
 
             <ul>
-              <li>The HTML <code>footer</code> element defines a <code>contentinfo</code> landmark when its context is the <code>body</code> element.</li>
+              <li>HTMLの<code>footer</code>要素は、そのコンテキストが<code>body</code>要素の場合、<code>contentinfo</code>ランドマークを定義します。</li>
```

Comment:
```suggestion
              <li>HTMLの<code>footer</code>要素は、そのコンテキストが<code>body</code>要素の場合、<code>contentinfo</code>ランドマークを明示します。</li>
```

Reply:
「明示」と「規定」どっちにしましょうかしら・・？

```suggestion
              <li>HTMLの<code>footer</code>要素は、そのコンテキストが<code>body</code>要素の場合、<code>contentinfo</code>ランドマークを規定します。</li>
```

Reply:
「明示」にします！